src/obfl-specification.html

```diff
@@ -1629,6 +1665,7 @@ <h4 id="L9241" class="include">toc-sequence</h4>
           <dd>A reference to a table-of-contents</dd>
         <dt>range [required]</dt>
           <dd>A data selection property (document/volume)</dd>
+          <dd>In a non-resumed range, the toc-entry-on-resumed TOC entries will be skipped.</dd>
```

Comment:
I think this is something you forgot to delete?

Reply:
Yup. 👍